client/src/components/ui/task-card.tsx

```diff
@@ -1,103 +1,74 @@
 import Image from "next/image";
-import React from "react";
+import Link from "next/link";
+import React, { useState } from "react";
 
-// please try to understand the "pricetype", not sure if we need to do useState for this part
+import { ChevronRightIcon } from "./icons"; // Adjust the import path as necessary
 
 // the props
 export interface TaskCardProps {
   id: string;
   state?: "BIDDING" | "EXPIRED" | "ONGOING" | "COMPLETED";
   category: string;
   title: string;
-  date: string;
-  location: string;
-  duration: string;
-  estimatePrice: string;
-  myOfferPrice: string;
-  priceType: "Estimated Price" | "My Offer";
+  description: string;
+  link?: string;
+  nestedContent?: React.ReactNode;
 }
 
-// the task card
-const TaskCard = ({
+const ProfileTag: React.FC<ProfileTagProps> = ({
```

Comment:
This is a repeat component in a different file? This file is for task-card not profile tag, I think the files were mixed up.

Reply:
This might be the source of the failed pushes.

Reply:
Think this is already resolved. my bad.